compiler/rustc_ast_passes/src/ast_validation.rs

```diff
@@ -764,9 +767,16 @@ fn validate_generic_param_order<'a>(
                     ordered_params += &pprust::bounds_to_string(&bounds);
                 }
             }
-            if let GenericParamKind::Type { default: Some(default) } = kind {
-                ordered_params += " = ";
-                ordered_params += &pprust::ty_to_string(default);
+            match kind {
+                GenericParamKind::Type { default: Some(default) } => {
+                    ordered_params += " = ";
+                    ordered_params += &pprust::ty_to_string(default);
+                }
+                GenericParamKind::Type { default: None } => (),
+                GenericParamKind::Lifetime => (),
+                // If/when const generic parameters are allowed to have defaults,
+                // add them to the output as is done for type defaults (above)
```

Comment:
```suggestion
                // FIXME(const_generics:defaults)
```
We've been using this `FIXME` to easily search for places where changes are necessary to support const generic defaults.